src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -115,6 +115,27 @@ protected function redirectToRoute(string $route, array $parameters = [], int $s
         return $this->redirect($this->generateUrl($route, $parameters), $status);
     }
 
+    /**
+     * Returns a RedirectResponse to last accessed url.
+     *
+     * @author Temuri Takalandze (ABGEO) <takalandzet@gmail.com>
+     *
+     * @final
+     */
+    protected function redirectToBack(): RedirectResponse
```

Comment:
`redirectBack` seems a better name to me (and is also the name in Rails btw, except that they use snake_case rather than camelCase). `to back` does not make sense in English.